Purchases/Public/RCPurchases.m

```diff
@@ -1134,6 +1134,28 @@ - (void)postReceiptWithTransaction:(SKPaymentTransaction *)transaction
                        }];
 }
 
+
```

Comment:
I'm curious why these methods have been added. Is this a side effect of splitting things out of `RCPurchases.m`?

Reply:
those two were in the `RCPurchases+SubscriberAttributes` file, which i deleted as part of this migration. it felt like they belonged in this class to me, but i can also pull them to the manager...

Reply:
OHhhhh that's fine.